device/cuda/src/seeding/track_params_estimation.cu

```diff
@@ -69,7 +82,7 @@ track_params_estimation::operator()(
     unsigned int num_blocks = seeds_size / num_threads + 1;
 
     // run the kernel
-    track_params_estimating_kernel<<<num_blocks, num_threads>>>(
+    kernels::estimate_track_params<<<num_blocks, num_threads>>>(
```

Comment:
Please remove the declaration of `traccc::cuda::track_params_estimating_kernel` from line 37. :stuck_out_tongue: